src/fluree/db/json_ld/api.cljc

```diff
@@ -122,6 +123,34 @@
    (promise-wrap
      (jld-ledger/load conn address))))
 
+(defn alias->address
+  "Returns a core.async channel with the connection-specific address of the
+  given ledger-alias."
+  [conn ledger-alias]
+  (conn-proto/-address conn ledger-alias nil))
+
+(defn load
+  "Loads an existing ledger by its alias (which will be converted to a
+  connection-specific address first)."
+  [conn ledger-alias]
+  (promise-wrap
+    (go
+      (let [address (<! (alias->address conn ledger-alias))]
+        (log/debug "Loading ledger from" address)
+        (<! (jld-ledger/load conn address))))))
+
+(defn load-if-exists
```

Comment:
Are all these `load`* functions meant to be part of the public api? It's not clear to me when I would use `load` vs `load-if-exists`.

Reply:
Yeah, there's definitely room for improvement here. Basically I needed a way to implement idempotent `create`. This was my "simplest possible thing that could possibly work" but I'd be happy to push it a little further in the direction of the ideal.